public_html/src/modules/Base/router/routes.js

```diff
@@ -1,7 +1,7 @@
 export default [
   {
     name: 'Base',
-    parent: 'App',
+    parent: 'Basic',
```

Comment:
Basic? It is not basic
why not App?
Root, Main etc Basic doesn't fit here I think